pkg/kf/commands/install/gke/gke.go

```diff
@@ -37,7 +37,8 @@ const (
 	numNodes              = "3"
 	defaultMaxPodsPerNode = "110"
 	addons                = "HorizontalPodAutoscaling,HttpLoadBalancing,Istio,CloudRun"
-	defaultGKEVersion     = "1.13.11-gke.11"
+	defaultReleaseChannel = "STABLE"
+	defaultGKEVersion     = "1.13.11-gke.14"
```

Comment:
nit: Is this still necessary, or does `defaultReleaseChannel` replace it?

Reply:
good catch, removed!